.gitignore

```diff
@@ -21,3 +21,4 @@ fineract-provider/src/main/resources/application.properties
 fineract-provider/out/
 fineract-provider/config/swagger/config.json
 fineract-provider/config/swagger/fineract-input.yaml
+fineract-client/config/swagger/templates/*.mustache
```

Comment:
@vidakovic this doesn't look right, is this intentional? Why?

Reply:
@vorburger It is. That way you can still give advanced users the option to add their own templates if they need something custom we don't support. There's also a disclaimer in the client doc that we don't manage any custom templates.

I've added that reading the previous doc where this was described as some kind of feature. If we don't want that then I can remove again.